Automattic/wp-calypso
# FSE: Remove page title block when editing template parts

#### Changes proposed in this Pull Request

* Remove the post title when editing a template part in FSE

#### Testing instructions

* Check out this branch and rebuild the FSE plugin for your local FSE test environment `npx lerna run dev --scope='@automattic/full-site-editing'`
* Edit a Header or Footer template part and check that the page title block does not display in the editor

**Before:**
<img width="630" alt="before" src="https://user-images.githubusercontent.com/3629020/62172956-4a995e00-b388-11e9-9219-2a95cf03050f.png">

**After**
<img width="632" alt="after" src="https://user-images.githubusercontent.com/3629020/62172966-52f19900-b388-11e9-84e7-b07db1b76de2.png">

Part of #34728
